homeassistant/components/deconz/sensor.py

```diff
@@ -158,6 +162,9 @@ def __init__(self, device, gateway):
         self._attr_state_class = STATE_CLASS.get(type(self._device))
         self._attr_unit_of_measurement = UNIT_OF_MEASUREMENT.get(type(self._device))
 
+        if device.type in Consumption.ZHATYPE:
+            self._attr_last_reset = dt_util.utc_from_timestamp(0)
```

Comment:
So all Zigbee devices never reset their consumption value?

Reply:
I asked Manuel and he said you need to reset the device